buildSrc/src/main/groovy/com/lesfurets/gradle/CopyDependenciesToMavenTreeTask.groovy

```diff
@@ -30,5 +30,6 @@ class CopyDependenciesToMavenTreeTask extends DefaultTask {
                 from artifact.file
             }
         }
+        //return getOutputDir()
```

Comment:
What's this all about?

Reply:
@nre-ableton  `CopyDependenciesToMavenTreeTask` copies `@Grab` dependencies to m2 cache,  we had some tests with `@Grab` usage 

But I can not find any test with `@Grapes or `@Grab` usage. 
I believe as we have no tests we can remove gradle build scripts and grape related configuration as well.

Reply:
Great, thanks for the cleanup.